README.md

```diff
@@ -1,4 +1,18 @@
-## devconfzm19
+Welcome to the Zambian Developer Conference official website for Facebook F8 2019.
+This website is an open source project, it has been made available to help provide:
+1. Details about the conference like;
+    - What the conference will be all about.
+    - Why the conference exists.
+    - What it is trying to achieve especially    where technology is concerned.
+    - Where it will be held and time.
+    - Who is eligible to attend.
+    - What happens after the conference.
+    - Contact details of the organizers.
+    - Help every developer out there especially newbies on how to develops and deploy a website on the internet.
 
+NB: SO ITS SAFE TO SAY THAT IT WILL ALSO BE A VERY BENEFICIAL PLATFORM TO LEARN FROM. 
 
+The website will be build on MySQL database being managed via PHP, with a great UI supported by JavaScript, HTLM and CSS. We hope that this open source project could be a great hands on opportunity for many developers out there especially the beginners, as we will develop and deploy a live website on the internet together as a community.
```

Comment:
@LuisRoyZulu06 to make it a bit more clear, we won't use PHP nor MySQL
You can replace the above paragraph with this:

**Technologies**
 
- React
- Bootstrap
- Netlify

Reply:
Okay thank you very for the clarification

Reply:
Let me know if you know clarity on anything else.